WMIAdventure/backend/WMIAdventure_backend/battle/businesslogic/BattleCard.py

```diff
@@ -1,15 +1,17 @@
-from typing import List
+from typing import List, Optional
 
 from battle.businesslogic.effects.Effect import Effect
 from battle.businesslogic.effects.EffectFactory import EffectFactory
 from cards.models import Card
 from .Buff import Buff
+from .buffs.CardDuplicatedBuff import CardDuplicatedBuff
 
 
 class BattleCard:
     """
 
     """
+    card_duplicated_buff: Optional[CardDuplicatedBuff]
```

Comment:
Optional daje mi jakieś flashbacki z Javy, w Pythonie 3.9 nie używa się czasami `CardDuplicatedBuff | None'?

Reply:
A nie, to jest z Pythona 3.10, a my korzystamy z 3.9